### PR TITLE
Commiting new tests to BranchEngineer

### PR DESCRIPTION
**Pull request by vishalkhade1508**
Hi,
I have commited Tests.. So plz review and approve.
Regards,
Vishal Khade

### CODE REVIEW

File: src/test/java/tests/TesterTest2.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	
+	@Test
+	public void test9()
+	{
+		Assert.assertTrue(true);
+		System.out.println("test9 passed");
+	}
+	
+	
+	@Test
```
Comment (vishal15khade) [inline src/test/java/tests/TesterTest2.java:17]: Hi Vishal,
Your code looks good...
Just add one more test at this point

**Review by vishal15khade — COMMENTED** [review]
Do the given work urgently

**Review by vishal15khade — CHANGES_REQUESTED** [review]
Do as requested

**Comment by vishalkhade1508** [comment]
Hi
I have added additional test case
Plz check......
Regards,
Vishal Khade

**Review by vishal15khade — APPROVED** [review]
Hi Vishal,
All good....
Now plz go ahead with merge with master branch...
Regards Team

**Pull request merged**